service/controller/v2/resources/vaultcrt/current.go

```diff
@@ -49,6 +50,12 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	// draining was done and the pods got removed we get an empty list here after
 	// the delete event got replayed. Then we just remove the secrets as usual.
 	if key.IsDeleted(customObject) {
+
+		if !r.checkCertType(customObject) {
+			r.logger.LogCtx(ctx, "level", "debug", "message", "do not supporting this cert type any longer.")
```

Comment:
Is "this cert type" a useful information?

```suggestion
			r.logger.LogCtx(ctx, "level", "debug", "message", "do not supporting this cert type any longer")
```

Reply:
If we see the message through luigi it will display as below, 
```
default/certconfigs/h0cg9-calico vaultcrtv2.GetCurrentState do not supporting this cert type any longer
```
I think it makes sense to arrange it with `this cert type` so you could guess it as `calico` type, WDYT?

Reply:
Ah yes because this reconciles the cert. That's fine then.